docker/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
`MIT` AFAIK is `GNU` compatible. Could it be used in TAO products?

Reply:
MIT in GPL2 is fine, see https://en.wikipedia.org/wiki/License_compatibility#Compatibility_of_FOSS_licenses